src/Controller/LevelEditorController.php

```diff
@@ -62,4 +62,20 @@ public function createLevel(): void
         $id = $levelManager->create($level, $cells);
         header('Location: /editor/edit?id=' . $id);
     }
+
+    public function delete(): void
+    {
+        $levelManager = new LevelManager();
+        $levels = $levelManager->selectAll('id');
+        $levelsId = array_column($levels, 'id');
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+            foreach ($_POST as $id => $value) {
+                $id = filter_var($id, FILTER_VALIDATE_INT);
```

Comment:
useless: the id comes from the database ;)

Reply:
id comes from post not from the database